apps/agreement/contracts/Agreement.sol

```diff
@@ -307,7 +307,7 @@ contract Agreement is IAgreement, AragonApp {
         emit ActionSubmitted(id, msg.sender);
 
         // Pay action submission fees
-        _payAppFees(currentSettingId, disputable, _submitter, id);
+        _payAppFees(id, action);
```

Comment:
Actually I'm not sure this is better! Now we're doing a lot more SLOADs from the `action` storage pointer.

In my comment, I meant to push the settings as a storage pointer—sorry if I was unclear!